### PR TITLE
Community page minor redesign, site modernisation

### PR DESCRIPTION
**Pull request by Blazzike**
image

### CODE REVIEW

**Comment by aikar** [comment]
"Modernization", man i can't keep up with "Is rounded corners cool again this year" with how many times that's flipped lol.

Overall looks good to me though.

File: src/community.twig [hunk 1]
```diff
@@ -33,52 +34,53 @@
...
+        <div class="main">
+          <div class="container">
+            <h1>Discord</h1>
+            <span>Discord is a popular option for many gamers to communicate with each other. We have a Discord community server that anyone can join.</span>
+          </div>
+        </div>
         <div class="container">
-          <h1>Discord</h1>
-          <span>Discord is a popular option for many gamers to communicate with each other. We have a Discord community server that anyone can join.</span>
+          <iframe src="https://discordapp.com/widget?id=289587909051416579&theme=dark" frameborder="0" allowtransparency="true"></iframe>
```
Comment (aikar) [inline src/community.twig]: can you fix this &theme to &amp;theme

Comment (Blazzike) [reply src/community.twig]: Ignore this: &amp; (testing)

Comment (Blazzike) [reply src/community.twig]: Okay, yeah sure. GitHub made that a little confusing.

Comment (aikar) [reply src/community.twig]: FYI it wasn't a discord thing, it was just invalid HTML, github made it clearer that it was.

Comment (Blazzike) [reply src/community.twig]: That's not the part I found slightly confusing, this was:
image

**Comment by hugmanrique** [comment]
Probs not 100% related, but the IRC font-awesome icon is missing some top margin on mobile (you can compare it with the forums icon)

File: src/community.twig (hunk 1 above)
Comment (aikar) [reply src/community.twig]: image
Github must not be encoding entities correctly.

**Comment by zachbr** [comment]
@hugmanrique I think(?) the old design did that too. Can you open an issue so no one forgets about it? Thanks

**Pull request merged**